o8g/Scripts/actions.py

```diff
@@ -1810,6 +1819,7 @@ def sealTokenCard(card, x = 0, y = 0, player = None):
     card.Subtype = 'Sealed'
     card.filter = "#99999999"
     card.controller = player
+    updateBlessCurse()
```

Comment:
The idea, why I didn't add this here was, that sealing a Blurse-Token, doesn't change the number of Blurse-tokens in play. So this update is unnecessary.